apps/solid-example/src/components/Header.tsx

```
@@ -0,0 +1,78 @@
+
+export default function Header() {
+  const [isOpen, setIsOpen] = createSignal(false)
+
+  return (
+    <>
+      <header class="p-4 flex items-center bg-gray-800 text-white shadow-lg">
+        <button
+          onClick={() => setIsOpen(true)}
+          class="p-2 hover:bg-gray-700 rounded-lg transition-colors"
+          aria-label="Open menu"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)